pkg/resources/execute_acceptance_test.go

```diff
@@ -20,17 +23,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestAcc_UnsafeExecute_basic(t *testing.T) {
-	id := acc.TestClient().Ids.RandomAccountObjectIdentifierWithPrefix("UNSAFE_EXECUTE_TEST_DATABASE_")
+func TestAcc_Execute_basic(t *testing.T) {
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifierWithPrefix("EXECUTE_TEST_DATABASE_")
```

Comment:
nit: (for after V1) I don't think we now need the prefixes with completely improved testing setup (these prefixes were valid more than a year ago)l we should get away from them